.env

```diff
@@ -1,5 +1,5 @@
-OPERATOR_SDK_VERSION=v1.39.0
-REVIEWERS=vivekr-splunk,rlieberman-splunk,patrykw-splunk,Igor-splunk,kasiakoziol
+OPERATOR_SDK_VERSION=v1.41.1
+REVIEWERS=vivekr-splunk,rlieberman-splunk,patrykw-splunk,Igor-splunk,kasiakoziol,gabrielm-splunk
 GO_VERSION=1.23.0
```

Comment:
Shouldn't we change this line too? 
Alternatively we could remove it. It's used only in github actions for setting up go and this can be replaced with direct reference to `go.mod`:
```
  - uses: actions/setup-go@v6
    with:
      go-version-file: 'path/to/go.mod'
```